modules/nf-core/cutesv/main.nf

```diff
@@ -0,0 +1,40 @@
+process CUTESV {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::cutesv=2.0.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/cutesv:1.0.12--pyhdfd78af_0' :
+        'quay.io/biocontainers/cutesv:2.0.2--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+    tuple val(meta2), path(fasta)
+
+    output:
+    tuple val(meta), path("*.vcf"), emit: vcf
+    path "versions.yml"                  , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def genotyping = params.enable_genotyping ? "--genotyping" : ''
```

Comment:
You can supply this flag using `task.ext.args` no need to make a separate argument for this

Reply:
I have removed this